plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -900,6 +911,20 @@ SecRule REQUEST_FILENAME "@rx /remote\.php/dav/calendars/(?:[^/]+/){2}[^/]+\.ics
             "t:none,\
             ctl:ruleRemoveById=200002"
 
+# Signing into caldav via iPhone
+SecRule REQUEST_FILENAME "@rx /remote.php/dav/calendars/[^/]+/[^/]+/$" \
+    "id:9508362,\
+    phase:1,\
+    pass,\
+    t:none,\
+    nolog,\
+    ver:'nextcloud-rule-exclusions-plugin/1.1.0',\
+    ctl:ruleRemoveTargetById=942430;XML:/*,\
+    ctl:ruleRemoveTargetById=942431;XML:/*,\
+    ctl:ruleRemoveTargetById=942432;XML:/*,\
+    ctl:ruleRemoveTargetById=942440;XML:/*,\
```

Comment:
Would be probably better to exclude whole XML (as you are doing it anyway), without xpath expression:
`ctl:ruleRemoveTargetById=942430;XML,\`